data/beaconrestapi/build.gradle

```diff
@@ -0,0 +1,21 @@
+dependencies {
+    compile 'io.javalin:javalin:3.5.0'
+    implementation project(':util')
+    implementation project(':data')
+    implementation project(':data:adapter')
+    implementation project(':ethereum:datastructures')
+    implementation project(':storage')
+    implementation project(':networking:p2p')
+    implementation project(':data:provider')
+
+    implementation 'org.apache.tuweni:tuweni-crypto'
+    implementation 'org.apache.tuweni:tuweni-units'
+    implementation 'com.google.guava:guava'
+    implementation 'org.apache.logging.log4j:log4j-api'
+    implementation 'org.apache.commons:commons-lang3'
+    runtime 'org.apache.logging.log4j:log4j-core'
+
+    test {
```

Comment:
This should probably be outside the `dependencies` section.

Reply:
done